docs/DeveloperGuide.md

```diff
@@ -5,13 +5,31 @@ title: Developer Guide
 
 ## **Introduction**
 
-### Purpose
+Welcome! We are excited that you want to learn about <strong><i>TBM</i></strong>.<br>
+This page contains step-by-step instructions on how to get started in contributing or getting yourself involved in this awesome project! However, before you get too excited, allow us to introduce you to the core concepts that underpins the heart of <strong><i>TBM</i></strong>.
```

Comment:
The comments by Team 2 advices to link Travelling BusinessMan to TBM, maybe we can do that here since it is the first mention of TBM?